packages/payment/src/services/payment-module.ts

```diff
@@ -160,305 +166,396 @@ export default class PaymentModuleService<
     )
   }
 
-  createPayment(
-    data: CreatePaymentDTO,
+  completePaymentCollections(
+    paymentCollectionId: string,
     sharedContext?: Context
-  ): Promise<PaymentDTO>
-  createPayment(
-    data: CreatePaymentDTO[],
+  ): Promise<PaymentCollectionDTO>
+  completePaymentCollections(
+    paymentCollectionId: string[],
     sharedContext?: Context
-  ): Promise<PaymentDTO[]>
+  ): Promise<PaymentCollectionDTO[]>
 
   @InjectTransactionManager("baseRepository_")
-  async createPayment(
-    data: CreatePaymentDTO | CreatePaymentDTO[],
+  async completePaymentCollections(
```

Comment:
thought(non-blocking): I wonder if we should just remove `completePaymentCollection` and let the full authorization be the completed state.

Reply:
I think that would make sense. ATM the method is just setting the timestamp on a payment collection which can be done implicitly upon full authorization as well.